plugins/frontmatter-validation/customParseFrontMatter.js

```diff
@@ -1,9 +1,78 @@
 // customParseFrontMatter.js
 const path = require('path');
+const fs = require('fs');
 
 // List to track files with issues
 let filesWithIssues = [];
 
+// Exceptions list configuration
+const EXCEPTIONS_FILE_PATH = path.join(process.cwd(), 'plugins/frontmatter-validation/frontmatter-exceptions.txt');
```

Comment:
added ability to have exceptions list for the frontmatter checker as `yarn docusaurus build` tries to build all locales. The translations docs don't yet have the slugs, we did the translations before we started enforcing this.